### ISSUE TITLE
exception on running `batali-tk`

### ISSUE THREAD

**Issue by gretel** [hw-labs/batali-tk#2]
```
$ bundle exec batali-tk
/Users/whoever/Sync/prjcts/xpl/kitchen/cookbooks/xpl_graphite_ng/vendor/cache/ruby/2.2.0/gems/batali-tk-0.2.0/bin/batali-tk:79:in `chdir': no implicit conversion of nil into String (TypeError)
  from /Users/whoever/Sync/prjcts/xpl/kitchen/cookbooks/xpl_graphite_ng/vendor/cache/ruby/2.2.0/gems/batali-tk-0.2.0/bin/batali-tk:79:in `<top (required)>'
  from /Users/whoever/Sync/prjcts/xpl/kitchen/cookbooks/xpl_graphite_ng/vendor/cache/ruby/2.2.0/bin/batali-tk:23:in `load'
  from /Users/whoever/Sync/prjcts/xpl/kitchen/cookbooks/xpl_graphite_ng/vendor/cache/ruby/2.2.0/bin/batali-tk:23:in `<main>'
```

**Comment by chrisroberts** [comment]
Hi. Thanks for the report! I have fixed this and released the update which is available in v0.2.2. Cheers!

**Comment by gretel** [comment]
works for me, thanks!
